Assets/MixedRealityToolkit-SDK/Features/UX/Interactable/Scripts/Layout/ButtonBorder.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX
+{
+    /// <summary>
+    /// Use a Cube 3D object as a border segment relative to the scale of the AnchorTransform
+    /// </summary>
+    [ExecuteInEditMode]
+    public class ButtonBorder : MonoBehaviour
+    {
+        [Tooltip("A pixel to Unity unit conversion, Default: 2048x2048 pixels covers a 1x1 Unity Unit or default primitive size")]
+        public float BasePixelScale = 2048;
```

Comment:
All fields should be private with public property accessors is needed.

Reply:
Will do, so everything is private with [SerializeField] tag?